packages/babel-plugin-proposal-decorators/README.md

```diff
@@ -62,11 +62,33 @@ Add the following line to your .babelrc file:
 }
 ```
 
-#### NOTE: Order of Plugins Matters!
+### Via CLI
+
+```sh
+babel --plugins @babel/plugin-proposal-decorators script.js
+```
+
+### Via Node API
+
+```javascript
+require("@babel/core").transform("code", {
+  plugins: ["@babel/plugin-proposal-decorators"]
+});
+```
+
+## Options
+
+### `legacy`
+
+`boolean`, defaults to `false`.
+
+Use the legacy (stage 1) decorators syntax and behavior.
```

Comment:
can do this later or skip ci commit - can mention this replaces the legacy plugin and link to it? (We should probably make a similar change in that readme about using this plugin if you are using v7)